src/types/FederationQueueState.ts

```diff
@@ -0,0 +1,20 @@
+// This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
+import type { ActivityId } from "./ActivityId";
+import type { InstanceId } from "./InstanceId";
+
+export type FederationQueueState = {
+  instance_id: InstanceId;
+  /**
+   * the last successfully sent activity id
+   */
+  last_successful_id?: ActivityId;
+  last_successful_published_time_at?: string;
+  /**
+   * how many failed attempts have been made to send the next activity
+   */
+  fail_count: number;
+  /**
+   * timestamp of the last retry attempt (when the last failing activity was resent)
+   */
+  last_retry_at?: string;
+};
```

Comment:
This is not part of the public api, its exposed throught a separate struct `ReadableFederationState`. So you can remove it.

Reply:
Even if I remove the export from the back end, it still exports it anyway because a higher up object needs it (even though it is getting flattened).

I'll have to add a special case to the script just for this file.